client/src/components/molecules/ViewScheduleMembersModal/index.tsx

```diff
@@ -1,23 +1,40 @@
 import classNames from 'classnames'
+import { useRouter } from 'next/router'
 import React, { FC, useState } from 'react'
+import { PulseLoader } from 'react-spinners'
 import { Coffee, Search, UserPlus } from 'react-feather'
 
 import MemberList from './MemberList'
 import Input from '~/components/atoms/Input'
+import useEmployeeSchedule from '~/hooks/useEmployeeSchedule'
 import AddScheduleMembersModal from './AddScheduleMembersModal'
 import ModalTemplate from '~/components/templates/ModalTemplate'
-import { scheduleMembers } from '~/utils/constants/dummyScheduleMembers'
 import ModalHeader from '~/components/templates/ModalTemplate/ModalHeader'
+import { IScheduleMember } from '~/utils/interfaces/scheduleMemberInterface'
 
 type Props = {
   isOpen: boolean
   closeModal: () => void
   scheduleName: string
 }
 
-const ViewScheduleMembersModal: FC<Props> = ({ isOpen, closeModal, scheduleName }): JSX.Element => {
+const ViewScheduleMembersModal: FC<Props> = (props): JSX.Element => {
+  const { isOpen, closeModal, scheduleName } = props
   const [isOpenAddNewSchedule, setIsOpenAddNewSchedule] = useState<boolean>(false)
   const [searchedVal, setSearchedVal] = useState<string>('')
+  const router = useRouter()
+  const { id } = router.query
+
+  const { getEmployeesByScheduleQuery } = useEmployeeSchedule()
+  const { data, isLoading } = getEmployeesByScheduleQuery(Number(id), isOpen)
+  const scheduleMembers = data?.employeesBySchedule
+
+  const filterMembers = (member: IScheduleMember, searchedVal: string): boolean => {
+    if (searchedVal.length === 0) {
+      return true
+    }
+    return member.name.toString().toLowerCase().includes(searchedVal.toString().toLowerCase())
```

Comment:
Try this code and check if it works as expected.

```suggestion
    if (searchedVal.length === 0) {
      return true;
    }
    
    const lowerCaseSearchedVal = String(searchedVal).toLowerCase();
    const lowerCaseName = member.name.toLowerCase();

    return lowerCaseName.includes(lowerCaseSearchedVal);
```